folly/futures/Future-inl.h

```diff
@@ -28,7 +28,7 @@
 #include <folly/futures/detail/Core.h>
 #include <folly/futures/Timekeeper.h>
 
-#if FOLLY_MOBILE || defined(__APPLE__)
+#if !HAVE_BOOST_CONTEXT || (FOLLY_MOBILE || defined(__APPLE__))
```

Comment:
Why's that? It already configures as !HAVE_BOOST_CONTEXT in Boost 1.61 and later because the configure check checks for things that folly uses that are no longer available in boost >= 1.61.

```
$ ./configure  | grep Context
checking whether the Boost::Context library is available... no
$ dpkg -l libboost-context-dev  | grep ^i
ii  libboost-context-dev:amd64 1.61.0.2     amd64        provides a sort of cooperative multitasking on a single thread (default version)
```